server/helpers/validation.ts

```diff
@@ -45,10 +45,8 @@ export const validatePost = (req: Request, res: Response, next: NextFunction) =>
     // See if the newly made Post is valid
     validate(newpost).then(errors => { // errors is an array of validation errors
         if (errors.length > 0) {
-            console.log("validation failed. errors: ", errors);
-            res.status(400).send(errors);
+            next(new Boom("Validation error", {statusCode : 400}));
```

Comment:
Rather use the Boom's helpers function. For example here `next(Boom.badRequest('Validation error');)`. Same thing in all places where you create Booms

Reply:
That definitely adds readability! Implementing the changes now!